applications/ParticleMechanicsApplication/tests/restart_tests.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+# Import Kratos
+import KratosMultiphysics
+
+# Import KratosUnittest
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+from KratosMultiphysics.ParticleMechanicsApplication import particle_mechanics_analysis
+
+import KratosMultiphysics.kratos_utilities as kratos_utils
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
```

Comment:
```suggestion
def GetFilePath(fileName):
    return str(pathlib.Path(__file__).absolute().parent / fileName)
```

please use a bit more modern python